nrcemt/qeels/gui/main_window.py

```diff
@@ -300,15 +332,19 @@ def detect(self):
         for item in self.width_array:
             width.append(item.width_var.get())
             checkbox.append(item.detect_var.get())
+
+        index = self.options.index(self.list.get(tk.ANCHOR))
```

Comment:
is there not a better way to do this?

Reply:
i can do something like this:
       index = self.list.curselection()
        ev = ev_vals[index[0]]

Reply:
yeah that would be better

Reply:
that way you don't need to assign the text values to a member variable at all